src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -142,6 +148,26 @@ internal class OktaApi : IOktaApi {
 		throw new BmxException( "Account could not be found" );
 	}
 
+	async Task<string> IOktaApi.GetMeResponseAsync( string sessionId ) {
+
+		try {
+			using var meRequest = new HttpRequestMessage( HttpMethod.Get, "users/me" );
+			using var meResponse = await _httpClient.SendAsync( meRequest );
```

Comment:
```suggestion
			using var meResponse = await _httpClient.GetAsync( "users/me" );
```

Had it like that because I wanted to look at some things in the request object. This fine with me too since we do it that way above

Reply:
Looks fine eliminating that variable